flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/avro/Datum.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.formats.parquet.avro;
+
+import java.io.Serializable;
+
+/** Test datum. */
+public class Datum implements Serializable {
```

Comment:
Does this even need to be serializable?

Reply:
It is copied from `AvroStreamingFileSinkITCase` and turned into a normal class, since it could be used in different test classes. I think, in general, it is not a bad idea to make it be serializable, since the data will be read from a parquet file.

Reply:
I guess in the end, it doesn't really matter since this is only in test scope.